src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,48 @@
+package study;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class SetTest {
+	private Set<Integer> numbers;
+
+	@BeforeEach
+	void setUp() {
+		numbers = new HashSet<>();
+		numbers.add(1);
+		numbers.add(1);
+		numbers.add(2);
+		numbers.add(3);
+	}
+
+	@DisplayName("size 확인")
+	@Test
+	void size() {
+		assertThat(numbers).hasSize(3);
+	}
+
+	@DisplayName("값 존재 확인")
+	@ParameterizedTest
+	@ValueSource(ints = {1, 2, 3})
+	void contains(int number) {
+		final boolean actual = numbers.contains(number);
+		assertThat(actual).isTrue();
```

Comment:
assertj 에서 제공해주는 `contains` 를 사용해보면 어떨까요?
```suggestion
		assertThat(numbers).contains(number);
```